### DISCUSSION TITLE
get_text method

### DISCUSSION THREAD

**Discussion by markokuz** [seleniumbase/SeleniumBase#2460] [General]
I'm trying to use the get_text method on "https://stockx.com/nike-blazer-mid-77-vintage-white-black" to get the element average sale price element text which is at the bottom of the page. But every time I try to use the get_text() method it says that the element is not present. Is there any work around?

**Comment by mdmintz** [comment]
This call found it:

```python
self.get_text('[data-component="MarketHistory"]')
```

``
'12-Month Historical\n12-Month Trade Range\n$25 - $309\nAll-Time Trade Range\n$75 - $96\nVolatility\n12%\nNumber of Sales\n8,403\nPrice Premium\n--\nAverage Sale Price\n$77'
``

You'll need to parse out the data you need from there.

You'll also need to use UC Mode to evade bot-detection on that site, and you may need to scroll to the bottom of the page first for that section to appear.
